doc/Programming-guide/Using-Kaa-endpoint-SDKs/index.md

```diff
@@ -6,1107 +6,19 @@ sort_idx: 40
 ---
 Using Kaa endpoint SDKs
 
+{% include variables.md %}
 
-* [Introduction](#introduction)
+* TOC
+{:toc}
 
-## Introduction
+Developing some IoT solution we all the time facing the same routine - creating network communication stack, log delivery functionality, event exchange between endpoints and etcetera.
+All of this functionality already provides Kaa platform and you can get it out of the box using Kaa endpoint SDK.
```

Comment:
I'd paraphrase the sentence. It is a little messy. 

> you can get it out of the box using Kaa endpoint SDK.
This part of sentence refers to the functionality, but in the sentence in refers to `Kaa platform`